### ISSUE TITLE
{Linked:Bug6782194} [Adaptive Cards> Designer]: Visual heading (Book time for learning) is not programmatically defined as heading.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8298]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25290.1010)
Browser Version: Version 111.0.1660.7 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL in edge dev browser.
2. Navigate to the 'Card Payload Editor' and press enter key to expand it.
3. Paste the attached JSON file.
4. Navigate to Preview mode button and press enter key to activate it.
5. Navigate to the 'Book time for learning' text and observe the issue.

**Actual Result:**  
'Book time for learning' text, is not defined as heading in open pane. Screen Reader simply announces as 'Book time for learning.

**Expected Result:** 
Screen Reader should announce as 'Book time for learning Heading'.

Text that is visually styled as a heading should also be programmatically implemented as a heading (h1, h2, etc.), In addition, the hierarchy of the headings should mirror the visual layout and use a logical progression. 

**User Impact:**  
Visually Impairment users who rely on screen reader will be unaware of the headings
Assistive technologies use headings to help people quickly orient their location within and navigation through a page. When headings are not implemented correctly it becomes much more difficult to locate relevant content.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B9C514377-76E5-4CFE-8300-DA3079D3CDC3%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships.docx&action=default&mobileredirect=true&share=IQF3Q1Gc5Xb-TIMA2jB5083DAcWOZyaXcx7TtNeRlXM4uHk

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/217776357-3e07eb51-30a5-4441-8687-a9506611c2da.mp4

[Json file.txt](https://github.com/microsoft/AdaptiveCards/files/10695812/Json.file.txt)

### Sample Code

Original tracked through below urL.

https://office.visualstudio.com/OC/_workitems/edit/6782194

**Comment by anna-dingler** [comment]
Hi @vagpt,

This issue can be resolved by updating the card payload. `TextBlock` has a `style` property that can be set to `heading`. Narrator will then announce it properly.

Let me know if you have any questions!

**Comment by anna-dingler** [comment]
Updated payload:

[Json.file (1).txt](https://github.com/microsoft/AdaptiveCards/files/11348752/Json.file.1.txt)

**Comment by vagpt** [comment]
This issue is working fine with the provided Json file above. Please find the attached video for the reference.


https://user-images.githubusercontent.com/95913748/235104537-4059f274-a72b-4225-998e-63935af2b77c.mp4
